### PR TITLE
docs: fix broken link to css variables doc

### PR DESCRIPTION
**Pull request by artmsilva**
## What does this PR do?

Fixes a broken link

## These changes will impact:

- [ ] commenters
- [ ] moderators
- [ ] admins
- [x] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

No.

### CODE REVIEW

File: docs/docs/css.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Here is a CSS example that modifies some of the CSS Variables.
 }
 ```
 
-You can find all CSS Variables using the Web Inspector or navigate to https://github.com/coralproject/talk/blob/main/CSS_VARIABLES.md for a comprehensive list and additional information.
+You can find all CSS Variables using the Web Inspector or navigate to [https://github.com/coralproject/talk/blob/main/CSS_VARIABLES.md](https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md) for a comprehensive list and additional information.
```
Comment (nick-funk) [inline docs/docs/css.md]: Redirecting from `https://github.com/coralproject/talk/blob/main/CSS_VARIABLES.md` to `https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md` might be confusing, would you mind making them match?

i.e.
```
[https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md](https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md)
```

**Review by nick-funk — CHANGES_REQUESTED** [review]
Thank you for helping contribute to Coral!

I've left a comment about your URL fix, have a look and let me know what you think!

File: docs/docs/css.md (hunk 1 above)
Comment (artmsilva) [reply docs/docs/css.md]:
```suggestion
You can find all CSS Variables using the Web Inspector or navigate to [https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md](https://github.com/coralproject/talk/blob/main/client/CSS_VARIABLES.md) for a comprehensive list and additional information.
```

**Review by nick-funk — APPROVED** [review]
Looks great, thank you for addressing the requested changes!

**Pull request merged**